internal/parser/src/imap/parser.h

```diff
@@ -14,12 +15,14 @@ struct ParseResult {
   // If not empty, contains the currently parsed command tag
   std::string tag;
   // Contains the protobuf serialized data
-  std::string command;
+  std::vector<uint8_t> command;
```

Comment:
can we align between `uint8_t` and `char` here and the methods below? or do we need to stick with `char` below for the C API?